Change box to bento/ubuntu-16.04 and make the box work on WSL

eliasylanen opened the pull request.

Commit:
Merge branch 'release/0.1.0'

* release/0.1.0:
  Update readme
  Add dummy-site
  Add main provision-file
  Add tools-playbooks
  Add system playbooks
  Add group_vars
  Add base Vagrantfile
  Add .gitignore
  Add base config-file
  Add license

Commit:
Merge branch 'release/0.1.1'

* release/0.1.1:
  Update readme

Commit:
Change box to bento/ubuntu-16.04

Commit:
Change box to bento/ubuntu-16.04

Commit:
Revert to original .gitignore

Commit:
Get stuff actually working on WSL

Commit:
Update README

Commit:
Merge branch 'develop'

Commit:
Fix conflicts

Commit:
Add stuff to .gitignore

villeristi commented on README.md:
```diff
@@ -19,13 +19,18 @@ Easily configurable local LEMP-environment for multiple hostnames.
 
 ### What if I run into problems...?
+* Sometimes the box has to be provisioned twice, since installing `yarn` occasionally throws an error
+* Insalling Mailcatcher has been disabled in `/provision/roles/tools/tasks/main.yml` for the time being, since it keeps failing
```
Correct typo

Jalle19 replied:
What error does yarn throw?

eliasylanen replied:
```TASK [tools : Node | Install Yarn] *********************************************
fatal: [default]: FAILED! => {"changed": true, "cmd": "curl -o- -L https://yarnpkg.com/install.sh | bash", "delta": "0:00:05.875026", "end": "2017-05-24 07:03:47.211845", "failed": true, "rc": 1, "start": "2017-0
5-24 07:03:41.336819", "stderr": "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n                                 Dload  Upload   Total   Spent    Left  Speed\n\r  0     0    0
   0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100  4978    0  4978    0     0   3705      0 --:--:--  0:00:01 --:-
-:--  3703\r100  6746    0  6746    0     0   5021      0 --:--:--  0:00:01 --:--:--  5019\n\n[1/2]: https://yarnpkg.com/latest.tar.gz --> /tmp/yarn.tar.gz.zW1uuSEQao\n  % Total    % Received % Xferd  Average Spe
ed   Time    Time     Time  Current\n                                 Dload  Upload   Total   Spent    Left  Speed\n\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100    92  100
    92    0     0    167      0 --:--:-- --:--:-- --:--:--   168\n\r  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0\r100   609    0   609    0     0    473      0 --:--:--  0:00:01
--:--:--  3668\n\r  0  760k    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0\r 35  760k   35  271k    0     0  91320      0  0:00:08  0:00:03  0:00:05  262k\r100  760k  100  760k    0     0
216k      0  0:00:03  0:00:03 --:--:--  509k\n\n[2/2]: https://yarnpkg.com/latest.tar.gz.asc --> /tmp/yarn.tar.gz.zW1uuSEQao.asc\n\r100    96  100    96    0     0    249      0 --:--:-- --:--:-- --:--:--   249\n
\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100   613    0   613    0     0   1153      0 --:--:-- --:--:-- --:--:--  299k\n\r100   900  100   900    0     0   1372      0 --
:--:-- --:--:-- --:--:--  1372\ngpg: keyring `/home/vagrant/.gnupg/secring.gpg' created\ngpg: key 86E50310: public key \"Yarn Packaging <yarn@dan.cx>\" imported\ngpg: Total number processed: 1\ngpg:
 imported: 1  (RSA: 1)\ngpg: Signature made Tue 23 May 2017 08:06:30 PM UTC using RSA key ID FD2497F5\ngpg: Good signature from \"Yarn Packaging <yarn@dan.cx>\"\ngpg: WARNING: This key is not certified with a tru
sted signature!\ngpg:          There is no indication that the signature belongs to the owner.\nPrimary key fingerprint: 72EC F46A 56B4 AD39 C907  BBB7 1646 B01B 86E5 0310\n     Subkey fingerprint: 6A01 0C51 6600
 6599 AA17  F081 46C2 130D FD24 97F5", "stderr_lines": ["  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current", "                                 Dload  Upload   Total   Spent    Left
Speed", "", "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0", "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0", "100  4978    0  4978    0     0   3
705      0 --:--:--  0:00:01 --:--:--  3703", "100  6746    0  6746    0     0   5021      0 --:--:--  0:00:01 --:--:--  5019", "", "[1/2]: https://yarnpkg.com/latest.tar.gz --> /tmp/yarn.tar.gz.zW1uuSEQao", "  %
 Total    % Received % Xferd  Average Speed   Time    Time     Time  Current", "                                 Dload  Upload   Total   Spent    Left  Speed", "", "  0     0    0     0    0     0      0      0 -
-:--:-- --:--:-- --:--:--     0", "100    92  100    92    0     0    167      0 --:--:-- --:--:-- --:--:--   168", "", "  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0", "100   609
    0   609    0     0    473      0 --:--:--  0:00:01 --:--:--  3668", "", "  0  760k    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0", " 35  760k   35  271k    0     0  91320      0  0:00:0
8  0:00:03  0:00:05  262k", "100  760k  100  760k    0     0   216k      0  0:00:03  0:00:03 --:--:--  509k", "", "[2/2]: https://yarnpkg.com/latest.tar.gz.asc --> /tmp/yarn.tar.gz.zW1uuSEQao.asc", "", "100    96
  100    96    0     0    249      0 --:--:-- --:--:-- --:--:--   249", "", "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0", "100   613    0   613    0     0   1153      0 --:--:-
- --:--:-- --:--:--  299k", "", "100   900  100   900    0     0   1372      0 --:--:-- --:--:-- --:--:--  1372", "gpg: keyring `/home/vagrant/.gnupg/secring.gpg' created", "gpg: key 86E50310: public key \"Yarn P
ackaging <yarn@dan.cx>\" imported", "gpg: Total number processed: 1", "gpg:               imported: 1  (RSA: 1)", "gpg: Signature made Tue 23 May 2017 08:06:30 PM UTC using RSA key ID FD2497F5", "gpg: Good signat
ure from \"Yarn Packaging <yarn@dan.cx>\"", "gpg: WARNING: This key is not certified with a trusted signature!", "gpg:          There is no indication that the signature belongs to the owner.", "Primary key finge
rprint: 72EC F46A 56B4 AD39 C907  BBB7 1646 B01B 86E5 0310", "     Subkey fingerprint: 6A01 0C51 6600 6599 AA17  F081 46C2 130D FD24 97F5"], "stdout": "\u001b[37mInstalling Yarn!\u001b[0m\n\u001b[36m> Downloading
 tarball...\u001b[0m\n\u001b[36m> Verifying integrity...\u001b[0m\n\u001b[32m> GPG signature looks good\u001b[0m\n\u001b[36m> Extracting to ~/.yarn...\u001b[0m\n\u001b[36m> Adding to $PATH...\u001b[0m\n\u001b[36m
> We've added the following to your /home/vagrant/.bashrc\n> If this isn't the profile of your current shell then please add the following to your correct profile:\n   \nexport PATH=\"$HOME/.yarn/bin:$PATH\"\n\u0
01b[0m\n\u001b[31m> Yarn was installed, but doesn't seem to be working :(.\u001b[0m", "stdout_lines": ["\u001b[37mInstalling Yarn!\u001b[0m", "\u001b[36m> Downloading tarball...\u001b[0m", "\u001b[36m> Verifying
integrity...\u001b[0m", "\u001b[32m> GPG signature looks good\u001b[0m", "\u001b[36m> Extracting to ~/.yarn...\u001b[0m", "\u001b[36m> Adding to $PATH...\u001b[0m", "\u001b[36m> We've added the following to your
/home/vagrant/.bashrc", "> If this isn't the profile of your current shell then please add the following to your correct profile:", "   ", "export PATH=\"$HOME/.yarn/bin:$PATH\"", "\u001b[0m", "\u001b[31m> Yarn w
as installed, but doesn't seem to be working :(.\u001b[0m"]}```

Jalle19 replied:
I would install it with `sudo npm install -g yarn` instead, that should at least not throw any odd errors and it won't attempt to place things in your home folder.

villeristi reviewed (changes requested):
There's something wrong with `nginx` as it doesn't resolve to correct folder defined with `sites-available/sitename.conf`. With another box this worked out-of-the-box.

Commit:
Fix Yarn install error and sites not showing


Merge remote-tracking branch 'origin/master'
Merge remote-tracking branch 'origin/master'

Commit:
Fix redirect loop on WP sites

Commit:
Re-add config.yml template

Commit:
Bring back ?$query_strings

Commit:
Revert to 'sites-enabled'

Commit:
Add template config.yml

Commit:
Update config.yml
